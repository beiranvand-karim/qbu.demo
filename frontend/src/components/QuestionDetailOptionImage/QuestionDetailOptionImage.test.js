/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {QuestionDetailOptionImage} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<QuestionDetailOptionImage />', () => {
   it('should render without crashing', () => {
      shallow(<QuestionDetailOptionImage/>)
   });
   it('should matches the snapshot', function () {
      const tree = renderer.create(<MemoryRouter><QuestionDetailOptionImage/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});