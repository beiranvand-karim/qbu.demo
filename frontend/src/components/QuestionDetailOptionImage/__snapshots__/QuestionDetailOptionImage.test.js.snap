// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuestionDetailOptionImage /> should matches the snapshot 1`] = `
<div
  className="position-relative form-check"
>
  <label
    className="form-check-label"
  >
    <label
      className=""
    >
      <input
        className="form-check-input"
        name="questionDetailRadio"
        onChange={[Function]}
        type="radio"
      />
      <img />
    </label>
  </label>
</div>
`;
